features/barclamp_ceph.feature

```diff
@@ -0,0 +1,33 @@
+@ceph
+Feature: Tests Ceph barclamp deployment
+  As an actor
+  I want to perform various validations
+  In order to verify the feature functionality
+
+  Background:
+    Given the chef role "ceph-mon" exists on admin node
+    And the chef role "ceph-mds" exists on admin node
+    And the chef role "ceph-osd" exists on admin node
+    And the chef role "ceph-radosgw" exists on admin node
+    And the chef role "ceph-calamari" exists on admin node
+    And the "ceph" cookbook exists on the admin node
+
+  Scenario: Ceph deployment and functioning
+    Given the barclamp proposal for "ceph" is deployed
+    When the node with "ceph-mon" role has been detected successfully
+    And the node with "ceph-osd" role has been detected successfully
+    And the node with "ceph-radosgw" role has been detected successfully
```

Comment:
All 3 previous steps are ceph specific, it's better the step to be like `the ceph node with "ceph-mon" role has been detected successfully` to keep the step description be limited to ceph. Please don't forget that the steps are unique and reusable within the whole testsuite and here they would clash with other similar nodes steps.